docs/development/heat_dissipation.md

```diff
@@ -1,11 +1,92 @@
+
+The calculations for thermal resistance and heat transfer are quite similar to those for voltage, current and resistance.
+
+The thermal resistance is given in °C/W, the temperature in °C and the power in W.
+When a power (equivalent to current) of 1W is flowing through a thermal resistance with 2°C/W (equivalent to electrical resistance), a temperature difference of 2°C is seen over the component (equivalent to voltage drop across a resistance). Like with electronic circuits, thermal resistances can be placed in series or in parallel.
+
+**Example:**
+
+A heat sink is connected to a MOSFET with a screw only (no heat paste or silikon conductor). The thermal resistance between air and heat sink is $R_{th,1}=6$°C/W and the connection with the MOSFET has a thermal resistance of $R_{th,2}=1$°C/W. The connection between the semiconductor ant the casing has a thermal resistance of $R_{th,3}=5$°C/W and the ambient temperature is 20°C. The power dissipated in the semiconductor is $P=5$W.
+
+The first temperature difference from the air to the heat sink can be calculated:
+
+$$\Delta T_1 = P \cdot R_{th, 1} = 5W \cdot 6^\circ C/W = 30^\circ C$$
+
+The next temperature differences are calculated accordingly:
+
+$$\Delta T_2 = P \cdot R_{th, 2} = 5W \cdot 1^\circ C/W = 5^\circ C \\
+\Delta T_3 = P \cdot R_{th, 3} = 5W \cdot 5^\circ C/W = 25^\circ C$$
+
+So beginning with the ambient temperature, the epected temperature inside the MOSFET is the sum of the differences across all thermal resistances:
+
+$$T_{trans} = T_{amb} + \Delta T_1 + \Delta T_2 + \Delta T_3 = 80^\circ C$$
+
+It is clear, that the temperature rises with the ambient temperature or with increased thermal resistance in the chain. While this method is not highly precise, it gives a good overview of the weakest links and can help to improve cooling. The goal should always be to keep the temperature as low as possible without exaggerated complexity and cost.
+
+### Passive cooling
+
+When passive cooling is used, the air flow (or any other liquid) is created simply by the density differences due to temperature gradients. The MOSFET can be either connected to the PCB where its surface is than passing the heat to the air passively or to heat sink. The heat sink has an optimized surface and good heat conducting characteristics to spread the heat fast and transfer it to the air over a large surface area. In both cases, the airflow is low and so is the amount of heat dissipated. For low energy appliances, this method is suitable, for larger power electronics, active cooling can be necessary.
+
+### Active Cooling
+
+With active cooling, a forced flow is generated over the surface of the PCB or heat sink. This can be done by simply blowing air with a fan or pumping a liquid. While a fan is mostly the cheapes and easiest to implement, using a water cooled system can be necessary for very high power density and with limited space.
```

Comment:
```suggestion
With active cooling, a forced flow is generated over the surface of the PCB or heat sink. This can be done by simply blowing air with a fan or pumping a liquid. While a fan is mostly cheapest and easiest to implement, using a water cooled system can be necessary for very high power density and with limited space.
```